ansible/deploy-tor.yml

```diff
@@ -3,35 +3,9 @@
 
 - hosts: have_tor
   gather_facts: true # to fetch `jessie` codename
+  roles:
+    - role: tor-bin
```

Comment:
So, this is a subroutine call in the Ansible world, eh?

Reply:
It's more like a package of stuff that allow you to configure or install some unit of functionality on a system.
See: https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html

Reply:
Yes, I usually treat it as a subroutine call :)
Making a "good" and "reusable" package is a complex exercise and sometimes it leads to (possibly, unnecessary) levels of abstractions.

Reply:
Thanks for the docs! Yeah, I figured it's a complex exercise!